test/data/testSP.py

```diff
@@ -241,19 +241,10 @@ def testmoments(self):
                 self.assertEquals(m, 0.0)
 
     @skipForParser('ADF', 'Does not support metadata yet')
-    @skipForParser('GAMESS', 'Does not support metadata yet')
     @skipForParser('GAMESSUK', 'Does not support metadata yet')
-    @skipForParser('Gaussian', 'Does not support metadata yet')
-    @skipForParser('Jaguar', 'Does not support metadata yet')
-    @skipForParser('Molpro', 'Does not support metadata yet')
-    @skipForParser('NWChem', 'Does not support metadata yet')
-    @skipForParser('ORCA', 'Does not support metadata yet')
-    @skipForParser('Psi', 'Does not support metadata yet')
-    @skipForParser('QChem', 'Does not support metadata yet')
     def testmetadata(self):
         """Does metadata have expected keys and values?"""
         self.assertTrue(hasattr(self.data, "metadata"))
-        self.assertIn("basis_set", self.data.metadata)
         self.assertIn("methods", self.data.metadata)
         self.assertIn("package", self.data.metadata)
         self.assertIn("package_version", self.data.metadata)

```

Comment:
I think so. As a string, `metadata['basis_set']` can contain whatever you want, and there are some common ways to specify split basis sets, etc. The intention of `metadata` was to have something to store extra information we don't yet parse or won't ever parse. It is not meant to backwards compatible.

We do parse the basis set parameters in `gbasis` so in a sense `metadata['basis_set']` is redundant. It is only a name.